test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -77,7 +77,8 @@ public void PushCommand_PushToFileSystemSourceUnixStyle()
         public void PushCommand_PushToFileSystemSourceUncStyle()
         {
             // UNC only works in Windows. So skip this test if we're running on Unix,
-            if (Path.DirectorySeparatorChar == '/')
+            if (Environment.OSVersion.Platform == PlatformID.MacOSX ||
+                  Environment.OSVersion.Platform == PlatformID.Unix)
```

Comment:
Going forward I'd suggest using a platform trait to indicate platform specific tests like this
```C#
[Fact]
[Trait("Platform", "WindowsOnly")]
public void PushCommand_PushToFileSystemSourceUncStyle()
{
...
}
```
Having the trait one can invoke xunit runner with ```-ExcludeTraits "Platform=WindowsOnly"```